<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员控制台</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="onlineList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>421</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>在线用户列表：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chatRoom">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>聊天室：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>管理员控制台</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
